src/style/text.rs

```diff
@@ -3,11 +3,26 @@ use super::font::{FontDesc, FontFamily, FontStyle, FontTransform};
 use super::size::{HasDimension, SizeDesc};
 use super::BLACK;
 
+/// The alignment of the text.
+///
+/// Used to determine the invariant (anchor) points for backends
+/// which render the text on the client side.
```

Comment:
Just feel it's better to have more things documented. Such as why this is useful, when we should use this, etc.

Reply:
I won't bother too much as I think that the proper implementation will be ready in few weeks, but I'll add few more details, no worries.